sample-apps/springboot/build.gradle.kts

```diff
@@ -14,27 +16,21 @@ dependencies {
 }
 
 jib {
-  to {
-    image = "public.ecr.aws/aws-otel-test/aws-otel-java-springboot"
+  configureImages(
+    "public.ecr.aws/aws-otel-test/aws-opentelemetry-java-base:alpha",
+    "public.ecr.aws/aws-otel-test/aws-otel-java-springboot",
+    rootProject.property("localDocker")!!.equals("true"),
```

Comment:
Excuse my gradle ignorance but is there a reason that these don't match the changes in other gradle build files by explicitly stating the parameter they are targetting? 
```
   localDocker = rootProject.property("localDocker")!!.equals("true"),
    multiPlatform = !rootProject.property("localDocker")!!.equals("true"),
```

Reply:
localDocker and multiPlatform applies to the source image. However there are cases where the we want to use a remote image but no multiPlatform. E.g.: for generating this image: aws-opentelemetry-java-base:alpha